summary.txt

```diff
@@ -0,0 +1,52 @@
+Results/observations for group00
+Christine Tseng, Zuzanna Balewski
+
+We developed two outlier detection methods (see README.md), which produced
+slightly different results. While individually the strategies may not be ideal,
+we believe volumes identified by both more likely to be true outliers (or at
+least less dependent on possible biases in our methods).
+
+It would be very time consuming and expensive to reject scans with any
+outlier volumes, so we suggest excluding scans with more than 2
+(non-consecutive) outlier volumes.
+
+For group00 data, our outlier detection identifies the following volumes:
+group00_sub02_run1.nii 1, 2, 3, 158
+group00_sub02_run2.nii 35
+group00_sub05_run1.nii 20, 21, 22, 23
+group00_sub06_run1.nii 1, 2
+group00_sub06_run2.nii 22, 23
+group00_sub08_run1.nii 50
+group00_sub08_run2.nii 105, 106, 108, 156, 157, 158
+
+Using our rejection criterion, the following scans should be excluded:
+
+-- group00_sub02_run1.nii: The outlier volumes fall near the beginning
+and end of the 160TR run. Since run2 for sub02 is not problematic, maybe the
+participant did not understand that she should remain still during the
+entire run (not just when performing the task). (It is also possible
+that the poor data quality is due to an error when the scanner starts and
+stops recording?)
+
+-- group00_sub05_run1.nii: The four outlier volumes are clustered together, so
+it's likely that the participant shifted and returned to her original position
+(vs. moving constantly for the 8 sec span).
+
+-- group00_sub08_run2.nii: The outlier volumes are scattered in the second half
+of the scan. The participant could have gotten sleepy or fidgety and stopped
+paying attention to the task.
+
+Independently of these outlier computations, we have two observations from
+visually inspecting these scans (scripts/vis_4d_movie.py):
+
+-- The top and bottom portions of the brain are cut off in
+some scans. This is most striking for sub02 and sub03, and noticeable in
+varying degrees for other participants. These scans do not necessarily need to
+be excluded, but possible inferences both within and between participants
+will be limited.
+
+-- Both scans for sub03 have a large dark area in the right temporal lobe.
```

Comment:
This is a classic dropout effect, caused by variations in the magnetic field in the complex bone architecture near the middle of the temporal lobe.   The variations in the magnetic field cause the protons in these voxels to dephase more quickly (they are spinning at different speeds) and therefore the signal drops off very quickly.